docs/getting-started/registering-infrastructure-providers.md

```diff
@@ -61,6 +63,12 @@ Then, we'll use the Featureform CLI to register them
 featureform apply providers.py
 ```
 
+## Updating Providers
+
+In the event a provider's credentials change, Featureform supports updating previously registered providers' configurations. To update a provider, simply make the necessary changes to `providers.py`, for example, and register them again using the CLI command.
```

Comment:
Nitpick: Language feels a bit clunky propose simplifying it, maybe like: 

"To update a previously registered provider's configuration, make the necessary changes to providers.py (for example) and register them again using the CLI command. 

The description field can always be updated, but which configuration fields are updatable depends on the provider type. "

Not sure if that flows better?

You can find the list of valid fields for each provider type on their specific page below.